bluesky/run_engine.py

```diff
@@ -415,13 +421,41 @@ def request_pause(self, defer=False, name=None, callback=None):
                 print("Cannot pause from {0} state. "
                       "Ignoring request.".format(self.state))
 
-    def _register_scan_callback(self, name, func):
-        """Register a callback to be processed by the scan thread.
+    def _subscribe_lossless(self, name, func):
```

Comment:
:+1: on clarifying the 'lossless' callbacks (but it's not really related to the PR and also not mentioned in your summary...)